frontend/src/store/modules/category/actions.js

```diff
@@ -32,5 +32,13 @@ export default {
 
     deleteCategoryTags: ({commit}) => {
         commit('DELETE_CATEGORY_TAGS');
-    }
+    },
+
+    fetchCategory: (context, categoryId) => {
```

Comment:
hm, all categories might be fetched `fetchAllCategories` previously, does it necessary `fetchCategory`?